controller.go

```diff
@@ -397,3 +402,14 @@ func (c *controller) isDriverGlobalScoped(networkType string) (bool, error) {
 func (c *controller) GC() {
 	sandbox.GC()
 }
+
+func (c *controller) scope(networkType string) (driverapi.Scope, error) {
```

Comment:
Addressed this. 
Add `DataScope() DataScope` in `datastore.KV` interface.
Add `dataScope   datastore.DataScope` in `network` struct and assign it in `controller.addNetwork`.
For KV objects like `bitseq.Handle`/`ipam.Allocator`, since they have reference to a single `datastore.DataStore` field to store states, `DataScope()` is useless for them, just return `datastore.DataScope(-1)` in `DataScope()`
For `overlay.network`, simply return `datastore.GlobalScope`.